src/CalcManager/pch.h

```diff
@@ -24,6 +22,26 @@
 #include <list>
 #include <regex>
 #include <unordered_map>
-#include <intsafe.h>
 #include <array>
+#include <string_view>
+
+#include "win_data_types_cross_platform.h"
+
+#if defined(_WIN32) && defined(_MSC_VER)
```

Comment:
I think logic like this should be moved into each `_cross_platform.h` file.  For example, `winerror_cross_platform.h` could have:
```C++
#if defined(_WIN32) && defined(_MSC_VER)
#include <winerror.h>
#else 
typedef int32_t HRESULT;
...
#endif 
```

Then, any file in the project that needs error code types can `#include winerror_cross_platform.h` and the right thing will happen.

Reply:
Fixed by 9a03400